backend/packages/Upgrade/src/api/repositories/LogRepository.ts

```diff
@@ -685,7 +687,7 @@ export class LogRepository extends Repository<Log> {
     isFactorialExperiment: boolean,
     unitOfAssignment: string
   ): SelectQueryBuilder<Experiment> {
-    const experimentRepo = getCustomRepository(ExperimentRepository, 'export');
+    const experimentRepo = Container.getCustomRepository(ExperimentRepository);
```

Comment:
For now, we need to continue to use the 'export' custom repository so that the log queries associated with exports are still performed on the replica.

Reply:
Okay, I thought we don't want that. I Will push the update.